### ISSUE TITLE
ResponsiveSquare

### ISSUE THREAD

**Issue by lucaguindani** [Ecodev/angular-natural-gallery#2]
How do you specify the gallery format in this Angular integration? To use for instance the ResponsiveSquare version like in the example below.

https://github.com/Ecodev/natural-gallery-js/blob/master/docs/responsive-square.html

Can't find it in the doc.

Thanks!

**Comment by sambaptista** [comment]
It's not implemented at all. Only the Natural format is for now.

**Comment by lucaguindani** [comment]
I was willing to try the square-responsive format to see if the following issue could be avoided. Sometimes images ratios are very different and it could lead to visual inconsistencies.

![masonry-issue](https://user-images.githubusercontent.com/1675064/46474368-f4e53b00-c7e2-11e8-81fb-cc0991f7fc3e.jpg)

**Comment by sambaptista** [comment]
It wouldn't be that worth it to implement a new format in this lib just for a trial.

As only the rendering is concerned, maybe you can use the original JSON into the core natural-gallery-js  in a demonstration page using the format you want... 

Maybe an option to force min-width on items ? the top and bottom of thin images would be shrinked.

**Comment by lucaguindani** [comment]
Yes a "min-width" option could be a great solution to avoid that issue even if the image is shrinked a little. Simply applying a min-width via css in the list component isn't working and breaks the masonry (seems logical if the items are computed beforehand).

**Comment by sambaptista** [comment]
https://github.com/Ecodev/natural-gallery-js/issues/21
